src/components/ContentHighlights/HighlightStepper/SelectContentSearchPagination.jsx

```diff
@@ -11,10 +11,12 @@ export const BaseSearchPagination = ({
   <ActionRow>
     <DataTable.RowStatus />
     <ActionRow.Spacer />
-    <Pagination.Reduced
+    <Pagination
+      variant="reduced"
       currentPage={currentRefinement}
-      handlePageSelect={(pageNum) => refine(pageNum)}
+      onPageSelect={(pageNum) => refine(pageNum)}
       pageCount={nbPages}
+      icons={{}}
```

Comment:
nit: it might be worth adding an inline comment here explaining why we're passing an empty obj here (i.e., to remove the default icons provided).